kubetest2-tf/deployer/deployer.go

```diff
@@ -103,7 +105,22 @@ func (d *deployer) initialize() error {
 	if err := d.checkDependencies(); err != nil {
 		return err
 	}
-	d.provider = powervs.PowerVSProvider
+	// Use the --target-provider flag or fallback to TARGET_PROVIDER environment variable
+	targetProvider := d.TargetProvider
+	if targetProvider == "" {
+		targetProvider = os.Getenv("TARGET_PROVIDER")
+	}
+	if targetProvider == "" {
+		return fmt.Errorf("provider not specified. Use --provider or set the TARGET_PROVIDER environment variable")
+	}
```

Comment:
lets set the default value as powervs and remove this check and also lets avoid reading from env for now.

Reply:
Yes, kept powervs as default